app/models/miq_shortcut.rb

```diff
@@ -20,8 +26,7 @@ def self.seed
       end
     end
 
-    all.each do |rec|
-      next if names.include?(rec.name)
+    db_data.each do |name, rec|
```

Comment:
Seems like this block is unnecessary as you are now doing the delete above?  Or am I reading this wrong?